packages/@aws-cdk/aws-efs/lib/efs-file-system.ts

```diff
@@ -199,17 +185,17 @@ export interface EfsFileSystemAttributes {
  *
  * @resource AWS::EFS::FileSystem
  */
-export class EfsFileSystem extends EfsFileSystemBase {
+export class FileSystem extends Resource implements IFileSystem {
 
   /**
    * Import an existing File System from the given properties.
    */
-  public static fromEfsFileSystemAttributes(scope: Construct, id: string, attrs: EfsFileSystemAttributes): IEfsFileSystem {
-    class Import extends EfsFileSystemBase implements IEfsFileSystem {
+  public static fromFileSystemAttributes(scope: Construct, id: string, attrs: FileSystemAttributes): IFileSystem {
+    class Import extends Resource implements IFileSystem {
       public readonly fileSystemId = attrs.fileSystemID;
```

Comment:
If this compiles, the property should be changed to `fileSystemId`.

Reply:
Of course, it'll compile. Because it's correct.